data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -638,33 +492,26 @@ private void generateLineageDot() {
 
 		StringBuilder middle = new StringBuilder();         // all nodes definitions
 		// input tables to override output tables if duplicates are found
-		for (Node node : finalOutTableNodes) {
-			middle.append("\n" + node.toDotString());
-		}
-		for (Node node : finalInTableNodes) {
-			middle.append("\n" + node.toDotString());
-		}
-		for (Node node : finalFunctions) {
-			middle.append("\n" + node.toDotString());
-		}
-		for (Node node : finalConstants) {
-			middle.append("\n" + node.toDotString());
-		}
+		middle.append("\n" + tableNode);
 		middle.append("\n");
-
+		//LOGGER.info("node id------"+tableNode.getNodeId());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)